cosmos/pulsar.json

```diff
@@ -0,0 +1,57 @@
+{
+  "nodeProvider": {
+    "name": "Stake Source",
+    "email": "taariq@stakesource.network"
+  },
+  "rpc": "https://rpc.pulsar3.scrttestnet.com",
+  "rest": "https://api.pulsar3.scrttestnet.com",
+  "chainId": "pulsar-3",
+  "chainName": "Secret Testnet",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pulsar/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "SCRT",
+    "coinMinimalDenom": "uscrt",
+    "coinDecimals": 6,
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pulsar/uscrt.png"
+  },
+  "walletUrl": "https://testnet.keplr.app/chains/secret-network",
+  "walletUrlForStaking": "https://testnet.keplr.app/chains/secret-network",
```

Comment:
@levackt Could you remove two field(`walletUrl`, `walletUrlForStaking`)?
These are invalid URLs.

Reply:
No problem, done